spec/controllers/two_factor_authentication/backup_code_verification_controller_spec.rb

```diff
@@ -20,27 +23,26 @@
   describe '#create' do
     context 'when the user enters a valid backup code' do
       it 'tracks the valid authentication event' do
-        sign_in_before_2fa
-
-        form = instance_double(BackupCodeVerificationForm)
-        response = FormResponse.new(
-          success: true, errors: {}, extra: { multi_factor_auth_method: 'backup_code' },
-        )
-        allow(BackupCodeVerificationForm).to receive(:new).
-          with(subject.current_user).and_return(form)
-        allow(form).to receive(:submit).and_return(response)
+        sign_in_before_2fa(user)
 
         stub_analytics
         stub_attempts_tracker
-        analytics_hash = { success: true, errors: {}, multi_factor_auth_method: 'backup_code' }
+        freeze_time do
+          analytics_hash = {
+            success: true,
+            errors: {},
+            multi_factor_auth_method: 'backup_code',
+            multi_factor_auth_method_created_at: Time.zone.now,
```

Comment:
Similar comment, this feels like it could be flakey, since the backup codes were created outside the freeze block?